### DISCUSSION TITLE
.orgue, .organ and folders?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#915] [General]
After trying GrandOrgueTool according to https://github.com/GrandOrgue/grandorgue/discussions/330 I am more than confused. 

From the wiki, I understand that a compressed (old format) sample-set could be extracted to any folder. To "introduce" it to GO you just open the .organ file from that folder. Right. (I store for instance the Kalvträsk sample-set in GrandOrgue/Organs).

From the wiki I also understand that .orgue sample-sets should be stored in the GrandOrgue/Organ\ packages and "introduced" by the Load command. But to be able to Load I first had to Install.

An ignorant guess is that by installing an .orgue sample set it is unpacked to some folder, but where under under which file name? Is this true also when you Open a .organ sample-set?

How do i "unload" a sample-set I do not use any more?

I stumble over .idx files in the GrandOrgueCache folder. What are these?

//Jens

**Comment by larspalo** [comment]
.orgue packages that are dropped into your OrganPackageDirectory (defined in the Defaults and Initial settings tab - Paths of Settings dialog) should be auto detected by GO and doesn't need to be installed. You will need to restart GO though for it to be picked up - this is likely what you've missed to do.

If you have the .orgue package in any other location then you must install it.

To remove an organ completely (if you really want to do it) you need to remove all files of it (sampleset on disk, settings, cache, etc) and un-register it (remove) in the Settings dialog Organs tab and Organ Packages tab (otherwise it will be reported as MISSING later). This is an area that definitely needs improvement in the GO management and there are already issues created for it but I'm working on other things right now.

**Reply by jason-barba** [reply]
You are perfectly right, missed the restart detail.
